docs/source/gbd2017_models/risk_attributable_causes/vitamin_a_deficiency/index.rst

```diff
@@ -209,6 +209,23 @@ The causes affected by the Vitamin A Deficiency risk in GBD 2017 include
     -
     - 3.65 (2.23 - 5.97)
     - No (only one study)
+	
+.. list-table:: Table 4.2: Pulled relative risks of GBD 2017 from GBD mapping 
+  :widths: 15 14 14 
+  :header-rows: 1
+
+  * - Cause
```

Comment:
In the same way we have the tables for the cause model, we should note where these relative risks come from at the draw level.  Risks are easy.  rrs come from the `relative_risk` source pulled with the rei_id.

Reply:
I'll add the data source note. Thanks

Reply:
I am confused about the difference between table 4.1 and 4.2. What are the different data sources and which one should we use? Do we know why they are different?

Reply:
Good question. Table 4.1 is from appendix of GBD lancet paper, table 4.2 are pulled data from Como